src/Metal/MTLCompat.cs

```diff
@@ -22,9 +23,13 @@ public static IMTLCounterSet[] GetIMTLCounterSets (this IMTLDevice This)
 			return NSArray.ArrayFromHandle<IMTLCounterSet>(global::ObjCRuntime.Messaging.IntPtr_objc_msgSend (This.Handle, Selector.GetHandle ("counterSets")));
 		}
 
+#if !NET
 		[Unavailable (PlatformName.iOS, PlatformArchitecture.All)]
 		[Unavailable (PlatformName.TvOS, PlatformArchitecture.All)]
```

Comment:
Could we remove this two? We are inside a #if MONOMAC. Since we are adding the change..

Reply:
If you are asking me, then that makes sense haha